code/modules/mob/living/simple_animal/hostile/mining_mobs/elites/elite.dm

```diff
@@ -173,46 +172,47 @@ While using this makes the system rely on OnFire, it still gives options for tim
 					activity = TUMOR_INACTIVE
 					activator = null
 
-				
-obj/structure/elite_tumor/proc/spawn_elite(var/mob/dead/observer/elitemind)
+/obj/structure/elite_tumor/proc/spawn_elite(mob/dead/observer/elitemind)
 	var/selectedspawn = pick(potentialspawns)
 	mychild = new selectedspawn(loc)
 	visible_message("<span class='boldwarning'>[mychild] emerges from [src]!</span>")
 	playsound(loc,'sound/effects/phasein.ogg', 200, 0, 50, TRUE, TRUE)
 	if(boosted)
 		mychild.key = elitemind.key
 		mychild.sentience_act()
+		notify_ghosts("[mychild.name] has been awakened in \the [get_area(src)]!", source = mychild, action = NOTIFY_ORBIT, flashwindow = FALSE, header = "Lavaland Elite awakened")
```

Comment:
`"[mychild.name] ..."` tends to produce non-grammatical sentences. Maybe you want `"\A [mychild] ..."`. Ditto below.

Reply:
Changed.